app/components/booking-list/booking-list.tsx

```diff
@@ -6,13 +6,13 @@ import { CalendarProvider, AgendaList, ExpandableCalendar } from "react-native-c
 import { GIVER_CASUAL_NAVY, SHADOW_1, WIDTH } from "#theme"
 import { BookingInfoCard } from "../booking-info-card/booking-info-card"
 import { PreBol16, PreReg12, PreReg14 } from "../basics/custom-texts/custom-texts"
-import { confirmedBookings } from "../../services/axios/confirmed-bookings"
+import { ConfirmedBookings } from "../../services/axios/confirmed-bookings"
 
 export interface BookingListProps {
   /**
    * 추가적인 padding, margin 을 줌으로써, 위치를 조정할 수 있습니다.
    */
-  bookings?: confirmedBookings[]
+  bookings?: ConfirmedBookings[]
```

Comment:
required prop 으로 바꾸세요. prop 주석을 달아주세요
```suggestion
  bookings: ConfirmedBookings[]
```